lang/MoarVM/Portfile

```diff
@@ -42,13 +33,18 @@ depends_build-append \
 # libatomic_ops:    https://trac.macports.org/ticket/53171
 depends_lib-append  port:dyncall \
                     port:libatomic_ops \
-                    port:libtommath \
-                    port:libuv
+                    port:libtommath
 
 patchfiles          patch-Configure.diff \
                     patch-dyncall.diff \
                     patch-setup.pm.diff
 
+if ![variant_isset macports_libuv] {
+patchfiles-append   patch-libuv-includes-path.diff \
+                    patch-libuv-legacy.diff \
+                    patch-moar.pc.diff
+}
```

Comment:
please add a link to upstream either to the commits/PRs that added those, or to the PR where you submitted them upstream.

Reply:
The patch for `libuv` is simply borrowed from the corresponding port. Discussion of it was in `libuv-devel` PR submitted by Herby; but anyway, it is pretty much identical to earlier `libuv` patches.
It has nothing to do with MoarVM upstream, it is a third-party software which they bundle.

Since Macports is stuck forever with an outdated `libuv`, as a solution for now using a bundled one works.

Other two are trivial are merely adjust paths where includes are placed, since by default MoarVM dumps those into `${prefix}/include/libuv`, which is confusing, since it is only meant for internal MoarVM and friends usage.

P. S. I spent some hours to make it work, it is not some randomly picked stuff over a coffee break :)